storage/oasis/nodeapi/file/consensus.go

```diff
@@ -81,7 +81,7 @@ func (c *FileConsensusApiLite) GetBlock(ctx context.Context, height int64) (*con
 func (c *FileConsensusApiLite) GetTransactionsWithResults(ctx context.Context, height int64) ([]nodeapi.TransactionWithResults, error) {
 	return GetSliceFromCacheOrCall(
 		c.db, height == consensus.HeightLatest,
-		generateCacheKey("GetTransactionsWithResults", height),
+		generateCacheKey("GetTransactionsWithResults.v2", height),
```

Comment:
neat

Reply:
:+1: 
can you add a comment that briefly explains why `.v2` is needed? e.g. `// Non-v2 cache key was used for values with heavily pruned roothash events`

Reply:
ya let's start keeping a history of these :shrug:

Reply:
snark :-1: 
but point well taken :+1:, taking back my request

Reply:
wasn't meant to be snarky sorry

Reply:
wait what even was the point that you took? isn't it good to have a comment here?

Reply:
oh hehe I totally misinterpreted then, sorry :facepalm:

The point I (apparently incorrectly) took was that it doesn't make sense to track the history of these tags because the cache is ephemeral and nobody needs to read/interpret the pre-v2 values any more.
Your call on whether to add the comment, I'm happy either way.

Reply:
anyway, adding